fc/protocol.py

```diff
@@ -670,10 +671,32 @@ def set_model(self, model):
                 model.add_equation(eq)
             # TODO: Check units of newly added equations; apply conversions where needed?
 
+            # Annotate state variables with the magic oxmeta:state_variable term
+            from cellmlmanip.rdf import create_rdf_node
+            is_version_of = create_rdf_node(('http://biomodels.net/biology-qualifiers/', 'isVersionOf'))
```

Comment:
"version" seems a weird way to annotate this? "x[i]" is a version of "x"? Or "V" is a version of a state variable?

Reply:
Yeah, it's weird, but was the closest match I could find without inventing our own property, so that's what pycml uses. 'V is a version of a state variable' is the sense of it. But could also be 'IKr is a version of a potassium current' or 'O1 is a version of an open state'.